usr/local/www/status_rrd_graph_settings.php

```diff
@@ -118,144 +115,108 @@
-					<b><?=gettext("Enables the RRD graphing backend.");?></b>
-				</td>
-			</tr>
-			<tr>
-                        	<td width="22%" valign="top" class="vtable"><?=gettext("Default category");?></td>
-	                        <td width="78%" class="vtable">
-					<select name="category" id="category" class="formselect" style="z-index: -10;" >
-					<?php
-					foreach ($categories as $category => $categoryd) {
-						echo "<option value=\"$category\"";
-						if ($category == $pconfig['category']) echo " selected=\"selected\"";
-						echo ">" . htmlspecialchars($categoryd) . "</option>\n";
-					}
-					?>
-					</select>
-					<b><?=gettext("This selects default category.");?></b>
-				</td>
-			</tr>
-			<tr>
-                        	<td width="22%" valign="top" class="vtable"><?=gettext("Default style");?></td>
-	                        <td width="78%" class="vtable">
-					<select name="style" class="formselect" style="z-index: -10;" >
-					<?php
-					foreach ($styles as $style => $styled) {
-						echo "<option value=\"$style\"";
-						if ($style == $pconfig['style']) echo " selected=\"selected\"";
-						echo ">" . htmlspecialchars($styled) . "</option>\n";
-					}
-					?>
-					</select>
-					<b><?=gettext("This selects the default style.");?></b>
-				</td>
-			</tr>
-			<tr>
-				<td width="22%" valign="top" class="vtable"><?=gettext("Default period");?></td>
-				<td width="78%" class="vtable">
-					<select name="period" class="formselect" style="z-index: -10;" >
-					<?php
-					foreach ($periods as $period => $periodd) {
-						echo "<option value=\"$period\"";
-						if ($period == $pconfig['period']) echo " selected=\"selected\"";
-						echo ">" . htmlspecialchars($periodd) . "</option>\n";
-					}
-					?>
-					</select>
-					<b><?=gettext("This selects the default period.");?></b>
-				</td>
-			</tr>
-			<tr>
-				<td width="22%" valign="top">&nbsp;</td>
-				<td width="78%">
-					<input name="Submit" type="submit" class="formbtn" value="<?=gettext("Save");?>" onclick="enable_change(true)" />
-				</td>
-			</tr>
-			<tr>
-				<td width="22%" valign="top">&nbsp;</td>
-				<td width="78%">
-					<input name="ResetRRD" type="submit" class="formbtn" value="<?=gettext("Reset RRD Data");?>" onclick="return confirm('<?=gettext('Do you really want to reset the RRD graphs? This will erase all graph data.');?>')" />
-				</td>
-			</tr>
-			<tr>
-				<td width="22%" height="53" valign="top">&nbsp;</td>
-				<td width="78%"><strong><span class="red"><?=gettext("Note:");?></span></strong><br />
-					<?=gettext("Graphs will not be allowed to be recreated within a 1 minute interval, please " .
-					"take this into account after changing the style.");?>
-				</td>
-			</tr>
-			</table>
-		</div>
-		</td>
-	</tr>
-</table>
-
-</form>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+$tab_array = array();
+$tabactive = ($curcat == "system") ? True:False;
```

Comment:
I noticed you kept this from the original file which is okay. This can however be rewritten to simply:
```php
$tab_array[] = array(gettext("System"), ($curcat == "system"), "status_rrd_graph.php?cat=system");
$tab_array[] = array(gettext("Traffic"), ($curcat == "traffic"), "status_rrd_graph.php?cat=traffic");
```